compose.override.example.yaml

```diff
@@ -18,16 +18,13 @@ services:
       - "${COMPOSER_CACHE_DIR:-${HOME}/.cache/composer}:/home/php/.composer/cache:z"
       - "${COMPOSER_HOME:-${HOME}/.composer}/auth.json:/home/php/.composer/auth.json:z"
       - "${COMPOSER_HOME:-${HOME}/.composer}/config.json:/home/php/.composer/config.json:z"
-      - ".docker-volume/starship.toml:/home/php/.config/starship.toml:rw,z" # allow for local edits to the integrated shell
-      - ".docker-volumes/atuin:/home/php/.local/share/atuin/:rw,z" # allow to keep the shell history stored in Atuin between container restarts
+      # allow for local edits to the integrated Starship shell customization
+      # Note: before adding this line make sure that the file exists (even empty), otherwise Docker will mount as a folder
```

Comment:
i am confused where this would need to exist. in the host system, relative to the project root?

and why is this one .docker-volume and then atuin .docker-volumes?

Reply:
yes, it should exist in the host system

the `s` at the end is something I fixed in another PR ... but would make more sense to more it here :) will do that

Reply:
pushed the code changes for this comment :+1: